packages/jest-runtime/src/index.ts

```diff
@@ -154,6 +156,7 @@ const supportsNodeColonModulePrefixInRequire = (() => {
 
 export default class Runtime {
   private readonly _cacheFS: Map<string, string>;
+  private readonly _cacheFSBuffer = new Map<string, Buffer>();
```

Comment:
@SimenB aren’t we doubling memory footprint by having these two caches? Most opened files will end up with an entry in both `_cacheFSBuffer` _and_ `_cacheFS`, while all reads will be from `_cacheFS` except the first one.

The impact can be insignificant, but there is a small chance that some tests may now fail with `JavaScript heap out of memory` in a new version. WDYT?